kornia/metrics/endpoint_error.py

```diff
@@ -0,0 +1,91 @@
+import torch
+from torch import Tensor, nn
+
+from kornia.core.check import KORNIA_CHECK, KORNIA_CHECK_IS_TENSOR, KORNIA_CHECK_SHAPE
+
+
+def aepe(input: torch.Tensor, target: torch.Tensor, reduction: str = "mean") -> torch.Tensor:
```

Comment:
i'd also add an alias `average_enpoint_error = aepe` (or viceversa) to have more explicit name functions

Reply:
+1

Reply:
@ChristophReich1996 can you add this alias?